packages/web3-utils/src/converters.ts

```diff
@@ -106,7 +98,10 @@ export const hexToBytes = (bytes: HexString): Buffer => bytesToBuffer(bytes);
  * Converts value to it's number representation
  */
 export const hexToNumber = (value: HexString): bigint | number => {
-	validateHexStringInput(value);
+	validator.validateJSONSchema(
+		{ type: 'array', items: [{ type: 'string', format: 'hexStrict' }] },
```

Comment:
why do we use `validateJSONSchema` instead of `validate(['hexstring', [value]])`?

Reply:
We should be using `validate(['hexstring', [value]])` (that's what a bulk of my comments were about above, looks like I missed this one, thank you!)